src/main/java/fr/delthas/skype/WebConnector.java

```diff
@@ -118,7 +124,7 @@ private User updateUser(JSONObject userJSON, boolean newContactType) throws Pars
     String userAvatarUrl = null;
     try {
       if (!newContactType) {
-        userUsername = userJSON.getString("username");
+        userUsername = userJSON.optString("username");
```

Comment:
This code will only be executed for the current user. If username is null, userUsername will be empty, which will set the current User username to the empty string. This will request "https://contacts.skype.com/contacts/v2/users//contacts" immediately after in WebConnector.java:93, which cannot work. Set this to getString maybe?